EIPS/eip-7405.md

```diff
@@ -0,0 +1,218 @@
+---
+eip: 7405
+title: Portable Smart Contract Accounts
+description: A solution working at the Proxy (ERC-1967) layer, allowing users to easily migrate their SCAs between different wallets.
+author: Aaron Yee (@aaronyee-eth)
+discussions-to: https://ethereum-magicians.org/t/erc-7405-portable-smart-contract-accounts/15236
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-07-26
+requires: 1967
```

Comment:
You mention EIP-191 in the specification section, so it probably should be here too.

Reply:
Okay